integration-tests/terraform/helm-windows/main.tf

```diff
@@ -196,9 +196,16 @@ resource "null_resource" "kubectl" {
   }
 }
 
+resource "time_sleep" "wait_15_min" {
+  depends_on = [helm_release.this]
+
+  create_duration = "15m"
```

Comment:
why 15 minutes? is that the same as we do for linux?

Reply:
What is typical for linux? My concern here is we are going to end up with really long test executions.
How do the execution times compare for lin and win?
Is there another method? Polling?  Presumably the windows image only changes every ~month or so caching should help?